plugins/spigot/src/main/java/com/managemc/spigot/command/handler/CmdReportsCreateHandler.java

```diff
@@ -0,0 +1,69 @@
+package com.managemc.spigot.command.handler;
+
+import com.managemc.spigot.command.handler.base.CommandHandlerAsync;
+import com.managemc.spigot.command.util.CommandAssertions;
+import com.managemc.spigot.command.util.ProcessedCommandArguments;
+import com.managemc.spigot.config.SpigotPluginConfig;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.Arrays;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class CmdReportsCreateHandler extends CommandHandlerAsync {
+
+  public static final String NO_SELF_REPORTS = "You cannot report yourself";
+  public static final String SUCCESS_MSG = ChatColor.GREEN + "Thank you for the report. If you have video " +
+      "evidence to add, please upload it to YouTube and submit it at https://managemc.com/reports.";
+  public static final String OFFLINE_MSG = ChatColor.YELLOW + "When possible, it is best to use this command while " +
+      "the player is still online and while you are recording them.";
+
+  private final SpigotPluginConfig config;
+
+  public CmdReportsCreateHandler(CommandSender sender, ProcessedCommandArguments args, SpigotPluginConfig config) {
+    super(sender, args);
+    this.config = config;
+  }
+
+  @Override
+  public void preProcessCommand() throws Exception {
+    playerSender = CommandAssertions.assertPlayerSender(sender);
+    CommandAssertions.assertArgsLengthAtLeast(args.getArgs(), 1);
```

Comment:
Throwing `CommandValidationException` at any point in the command processing lifecycle will cause processing to immediately terminate, providing the error message to the user and (optionally) the expected syntax. Feel free to add any reusable validations to `CommandAssertions`.